README.md

```diff
@@ -1,2 +1,4 @@
 # Db Query
 Simple PHP OOP library for creating database queries.
+ 
+ Run tests using: composer test
```

Comment:
Documentation should be updated properly. Introduce new section Contribution, sub section Development and subsub section Testing.
Need to add description hav to setup testing - e.g. copy codeception.yml-dist to codeception.yml, etc.